ansible_base/lib/serializers/common.py

```diff
@@ -11,7 +11,6 @@
 
 
 class CommonModelSerializer(serializers.ModelSerializer):
-    show_capabilities = ['edit', 'delete']
```

Comment:
I developed this originally. It is not my best. Please please please erase all vestiges of it in any new libraries.